tribe-ext-show-cost-field/index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Plugin Name:     Event Tickets Plus Extension: Show Cost Fields
+ * Description:     Event Tickets Plus hides the basic cost fields when activated, and automatically overrides the info with the costs of your tickets. When this extension is activated alongside Event Tickets Plus it will restore the hidden fields, allowing you to manually set the data.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Show_Cost_Field
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+// Do not load unless Tribe Common is fully loaded.
```

Comment:
Would recommend a space here. So instead of this:
```
}
// Do not load unless Tribe Common is fully loaded.
```
Maybe this:
```
}

// Do not load unless Tribe Common is fully loaded.
```